Midstone/ComponentFramework/SceneGameOver.cpp

```diff
@@ -0,0 +1,163 @@
+#include <glew.h>
+#include <iostream>
+#include <SDL.h>
+#include "SceneGameOver.h"
+#include <MMath.h>
+#include <QMath.h>
+#include <VMath.h>
+#include "Debug.h"
+#include "Mesh.h"
+#include "Shader.h"
+#include "Body.h"
+#include "stb_image.h"
+
+static bool LoadTextureFromMemory(const void* data, size_t data_size, GLuint* out_texture, int* out_width, int* out_height)
+{
+	// Load from file
+	int image_width = 0;
+	int image_height = 0;
+	unsigned char* image_data = stbi_load_from_memory((const unsigned char*)data, (int)data_size, &image_width, &image_height, NULL, 4);
+	if (image_data == NULL)
+		return false;
+
+	// Create a OpenGL texture identifier
+	GLuint image_texture;
+	glGenTextures(1, &image_texture);
+	glBindTexture(GL_TEXTURE_2D, image_texture);
+
+	// Setup filtering parameters for display
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
+
+	// Upload pixels into texture
+	glPixelStorei(GL_UNPACK_ROW_LENGTH, 0);
+	glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA, image_width, image_height, 0, GL_RGBA, GL_UNSIGNED_BYTE, image_data);
+	stbi_image_free(image_data);
+
+	*out_texture = image_texture;
+	*out_width = image_width;
+	*out_height = image_height;
+
+	return true;
+}
+
+// Open and read a file, then forward to LoadTextureFromMemory()
+static bool LoadTextureFromFile(const char* file_name, GLuint* out_texture, int* out_width, int* out_height)
+{
+	FILE* f;
+	errno_t err;
+	if ((err = fopen_s(&f, file_name, "rb")) != 0)
+	{
```

Comment:
Why is this empty?

Reply:
That i dunno i added only one line in that sceneui2